compiler/mx.compiler/suite.py

```diff
@@ -5,7 +5,7 @@
 
   "groupId" : "org.graalvm.compiler",
   "version" : "23.1.8.0",
-  "release" : False,
+  "release" : True,
```

Comment:
This "implies" that after merging this we will directly create the 23.1.8.0 tag. Is that indeed tha plen?